R/select_vpfiles.R

```diff
@@ -19,14 +19,15 @@
 #' @examples
 #' my_path <- "~/my/directory/"
 #' select_vpfiles(my_path, "2016-10-01", "2017-01-31", c("be"))
-select_vpfiles <- function(path, start_date, end_date,
-                           country = NULL, radar = NULL) {
+select_vpfiles <- function(path, date_min, date_max, country = NULL,
```

Comment:
Shouldn't this be directory instead of path?

Reply:
yes you're right, directory is better as we use that elsewhere as well